babybear/shuffleboard/BearMaxShuffleBoard.json

```diff
@@ -0,0 +1,322 @@
+        "titleType": 0,
+        "tiles": {
+          "0,3": {
+            "size": [
+              3,
+              3
+            ],
+            "content": {
+              "_type": "Graph",
+              "_source0": "network_table:///SmartDashboard/EveryBotMotorOvercurrentProtection",
+              "_title": "SmartDashboard/EveryBotMotorOvercurrentProtection",
+              "_glyph": 148,
+              "_showGlyph": false,
+              "Graph/Visible time": 30.0,
+              "Graph/X-axis auto scrolling": true,
+              "Y-axis/Automatic bounds": true,
+              "Y-axis/Upper bound": 1.0,
+              "Y-axis/Lower bound": -1.0,
+              "Y-axis/Unit": "ul",
+              "Visible data/SmartDashboard/EveryBotMotorOvercurrentProtection": true
+            }
+          },
+          "0,2": {
+            "size": [
+              2,
+              1
+            ],
+            "content": {
+              "_type": "Text View",
+              "_source0": "network_table:///SmartDashboard/ExtensionHorizontalArmMeters",
+              "_title": "SmartDashboard/ExtensionHorizontalArmMeters",
+              "_glyph": 148,
+              "_showGlyph": false
+            }
+          },
+          "2,2": {
+            "size": [
+              2,
+              1
+            ],
+            "content": {
+              "_type": "Text View",
+              "_source0": "network_table:///SmartDashboard/ExtensionVerticalArmMeters",
+              "_title": "SmartDashboard/ExtensionVerticalArmMeters",
+              "_glyph": 148,
+              "_showGlyph": false
+            }
+          },
+          "0,1": {
+            "size": [
+              2,
+              1
+            ],
+            "content": {
+              "_type": "Boolean Box",
+              "_source0": "network_table:///SmartDashboard/VerticalArmMiddleSensor",
+              "_title": "SmartDashboard/VerticalArmMiddleSensor",
+              "_glyph": 148,
+              "_showGlyph": false,
+              "Colors/Color when true": "#7CFC00FF",
+              "Colors/Color when false": "#8B0000FF"
+            }
+          },
+          "0,0": {
+            "size": [
+              2,
+              1
+            ],
+            "content": {
+              "_type": "Boolean Box",
+              "_source0": "network_table:///SmartDashboard/HorizontalArmSensor",
+              "_title": "SmartDashboard/HorizontalArmSensor",
+              "_glyph": 148,
+              "_showGlyph": false,
+              "Colors/Color when true": "#7CFC00FF",
+              "Colors/Color when false": "#8B0000FF"
+            }
+          },
+          "2,0": {
+            "size": [
+              2,
+              1
+            ],
+            "content": {
+              "_type": "Boolean Box",
+              "_source0": "network_table:///SmartDashboard/HorizontalArmSensorEncoderEverReset",
+              "_title": "SmartDashboard/HorizontalArmSensorEncoderEverReset",
```

Comment:
All of these arm things seem out of place for the robot I understand will be built for october.

Reply:
removed this section of the JSON.